package.json

```diff
@@ -732,9 +732,14 @@
 						"string",
 						"null"
 					],
-					"description": "Command to execute or full path to folder and executable to start the Mongo shell, needed by some Mongo scrapbook commands. If empty, will search in the system path for 'mongo'.",
+					"description": "Full path to folder and executable to start the Mongo shell, needed by some Mongo scrapbook commands. If empty, will search in the system path for 'mongo'.",
 					"default": null
 				},
+				"mongo.shell.args": {
+					"type": "string",
```

Comment:
Can we just make this a string array? So that we don't have to split the args ourselves?